test/test_ev.py

```diff
@@ -67,6 +67,7 @@ def test_validate_redraw_screen_enqueue(self):
             ui=self.mock_ui, config=self.conf,
             loop=self.mock_loop)
         dc.initialize = MagicMock()
+        dc.juju_state = MagicMock()
```

Comment:
ok so I think that's something that should be handled in `wait_for_deployed_services_ready`. Controller supports having juju_state set to None, so `wait_for_deployed_services_ready` should just return if there's no juju_state.

see e.g.: https://github.com/Ubuntu-Solutions-Engineering/openstack-installer/blob/patch-all-agents-started-v2/cloudinstall/core.py#L102